src/components/ModalWindow/ModalWindow.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Modal from 'react-bootstrap/Modal';
+import { useAppDispatch } from '../../hooks/hooks';
+import { addModalTodo } from '../../store/reducers/TodosSlice';
+import s from "./ModalWindow.module.scss";
+
+const ModalWindow: React.FC = () => {
+
+  const [input, setInput] = useState<any>({});
```

Comment:
type "any"

Reply:
Yes, I added interface first but for some reason it didn't want to work through this I had to add 'any' after many tries

Reply:
make an initial state then, coz empty object is not your input state, or read how to use "as" {} as InputState

Reply:
I made initial state, it works